app/server-utils/stampy.ts

```diff
@@ -94,15 +94,14 @@ const CODA_DOC_ID = 'fau7sl2hmG'
 // Use table ide, rather than names, in case of renames
 const QUESTION_DETAILS_TABLE = 'grid-sync-1059-File' // Answers
 const INITIAL_QUESTIONS_TABLE = 'table-yWog6qRzV4' // Initial questions
-const ON_SITE_TABLE = 'table-1Q8_MjxUes' // On-site answers
+const ON_SITE_TABLE = 'table-aOTSHIz_mN' // On-site answers
```

Comment:
I'm not sure, but I think this was causing the hanging "loading..." - the "On site" table doesn't have the text nor tags - the "On site API" table should be used, which has the same filters applied, but also returns the text

Reply:
huh,, this sounds like it might solve the race condition, let's see if people will still be able to randomly reproduce afterwards (hopefully not 🍀)